packages/tools/kadena-cli/src/account/commands/accountAddManual.ts

```diff
@@ -0,0 +1,41 @@
+import chalk from 'chalk';
+import type { Command } from 'commander';
+import debug from 'debug';
+import { assertCommandError } from '../../utils/command.util.js';
+import { createCommand } from '../../utils/createCommand.js';
+import { globalOptions } from '../../utils/globalOptions.js';
+import { addAccount } from '../utils/addAccount.js';
+import {
+  displayAddAccountSuccess,
+  overridePromptCb,
+} from '../utils/addHelpers.js';
+
+export const createAddAccountManualCommand: (
+  program: Command,
+  version: string,
+) => void = createCommand(
+  'add-manual',
+  'Add an existing account to the CLI',
+  [
+    globalOptions.accountAlias(),
+    globalOptions.accountName(),
+    globalOptions.fungible(),
+    globalOptions.network(),
+    globalOptions.chainId(),
+    globalOptions.publicKeys(),
+    globalOptions.predicate(),
+  ],
+
+  async (config): Promise<void> => {
+    try {
```

Comment:
`try {} catch(){}` is not needed, you may have copied it from other commands but we need to remove it there too, `createCommand` already handles this